### PR TITLE
Add tip on how to change orientation to landscape.

### PR DESCRIPTION
**Pull request by ScottBReynolds**

### CODE REVIEW

File: app/views/pages/developer-guide/user-uploads/generating-uploading-pdf.liquid [hunk 1]
```diff
@@ -106,6 +106,7 @@ query {
   }
 }
 ```
+{% include 'alert/tip', content: 'To change the PDF's orientation consider setting the page size in your css to landscape: @page { size: landscape; }' %}
```
Comment (diana-lakatos) [inline app/views/pages/developer-guide/user-uploads/generating-uploading-pdf.liquid]: Thank you @ScottBReynolds! What would you think about writing it like this:
"To change the PDF's orientation, set the page size..."

**Review by diana-lakatos — CHANGES_REQUESTED** [review]

File: app/views/pages/developer-guide/user-uploads/generating-uploading-pdf.liquid (hunk 1 above)
Comment (ScottBReynolds) [reply app/views/pages/developer-guide/user-uploads/generating-uploading-pdf.liquid]: Sounds good!

**Review by diana-lakatos — APPROVED** [review]

**Comment by diana-lakatos** [comment]
@pavelloz Please review, thank you!

File: app/views/pages/developer-guide/user-uploads/generating-uploading-pdf.liquid [hunk 2]
```diff
@@ -106,6 +106,7 @@ query {
   }
 }
 ```
+{% include 'alert/tip', content: 'To change the PDF's orientation, set the page size in your css to landscape: @page { size: landscape; }' %}
```
Comment (pavelloz) [inline app/views/pages/developer-guide/user-uploads/generating-uploading-pdf.liquid]:
```suggestion
{% include 'alert/tip', content: "To change the PDF's orientation, set the page size in your css to landscape: @page { size: landscape; }" %}
```

**Review by pavelloz — APPROVED** [review]

**Comment by pavelloz** [comment]
<3

**Pull request merged**